eZ/Publish/Core/Persistence/Legacy/Content/Gateway.php

```diff
@@ -18,78 +16,79 @@
 
 /**
  * Base class for content gateways.
+ *
+ * @internal For internal use by Persistence Handlers.
  */
 abstract class Gateway
 {
-    /**
-     * Get context definition for external storage layers.
-     *
-     * @return array
-     */
-    abstract public function getContext();
+    public const CONTENT_ITEM_TABLE = 'ezcontentobject';
+    public const CONTENT_NAME_TABLE = 'ezcontentobject_name';
+    public const CONTENT_FIELD_TABLE = 'ezcontentobject_attribute';
+    public const CONTENT_VERSION_TABLE = 'ezcontentobject_version';
+    public const CONTENT_RELATION_TABLE = 'ezcontentobject_link';
+
+    public const CONTENT_ITEM_SEQ = 'ezcontentobject_id_seq';
+    public const CONTENT_VERSION_SEQ = 'ezcontentobject_version_id_seq';
+    public const CONTENT_FIELD_SEQ = 'ezcontentobject_attribute_id_seq';
+    public const CONTENT_RELATION_SEQ = 'ezcontentobject_link_id_seq';
 
     /**
-     * Inserts a new content object.
-     *
-     * @param \eZ\Publish\SPI\Persistence\Content\CreateStruct $struct
-     * @param mixed $currentVersionNo
+     * Insert a new Content item.
      *
      * @return int ID
```

Comment:
```suggestion

```

Reply:
I'm gonna disagree here. This doc provides more value than the returned type. What if it returned something else than the new ID?

Reply:
> I'm gonna disagree here. This doc provides more value/information than the returned type. What if it returned something else than the new ID?

After internal sync I've reconsidered. Let's assume that `int` returned by `insert*` method is implicitly known as an ID from a database. Fixed via 6b9822bf.